test/db/cmd/cmd_tc

```diff
@@ -291,15 +291,15 @@ CMDS=ic*
 EXPECT=<<EOF
 fs classes
 "f class.public.Employee @ 0x1000079a8"
-"f field.Employee.isa @ 0x00000000"
+"f field.Employee.isa @ 0xffffffffffffffff"
```

Comment:
Is this change intended?

Reply:
yes, the previous value was set by `RZ_NEW0` this one is because the default value is `UT64_MAX`